assets/js/utils/headerUtils.ts

```diff
@@ -0,0 +1,47 @@
+import type { EventInfo } from "../components/Header";
+import { Presence } from "phoenix";
+import { getChannelId } from "../utils/channelUtils";
+import axios from "../services/index";
+
+export const initEventInfo = (): EventInfo => {
+  return {
+    link: "",
+    title: "",
+    description: "",
+    start_date: "",
+    presenters: [],
+  };
+};
+
+export const getEventInfo = (setEventInfo: React.Dispatch<React.SetStateAction<EventInfo>>) => {
+  axios
+    .get("/resources/webinars/" + getChannelId())
+    .then((response) => {
+      console.log(response.data);
+      const start_date = response.data.webinar.start_date.replace("T", " ");
+      setEventInfo({ ...response.data.webinar, start_date: start_date });
+    })
+    .catch(() => {
+      alert("Couldn't get event information. Please reload this page.");
+    });
+};
+
+export const syncParticipantsNumber = (eventChannel, setParticipantsNumber) => {
+  if (eventChannel) {
+    const presence = new Presence(eventChannel);
+
+    const updateParticipantsNumber = () => {
+      let counter = 0;
+      presence.list((name: string, metas: any) => {
+        counter += 1;
+      });
+      setParticipantsNumber(counter);
+    };
```

Comment:
I think this way it is simpler.
```suggestion
    const updateParticipantsNumber = () => {
      setParticipantsNumber(presence.list().length);
    };
```